### PR TITLE
resolving issue #4

### PR DESCRIPTION
**Pull request by needToRoll**
small changes in the  readme according to #4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,3 +79,10 @@ The you can run `ng serve` and visit the application in the browser on `localhos
 You can also enable the production version with `ng serve -prod`.
 
 For more detailed information on the CLI visit their [GitHub](https://github.com/angular/angular-cli) page, go to the README.md in the frontend project or just ask Mario Kunz (mario99.kunz@gmail.com).
+
+#GIT
+
+##Pushing and Pull requests
+
+Minor changes (change on one file, less than 5 lines and without bigger influence) can be pushed to the master branch directly
```
Comment (jmesserli) [inline README.md]: Please add a dot.

File: README.md [hunk 2]
```diff
@@ -79,3 +79,10 @@ The you can run `ng serve` and visit the application in the browser on `localhos
...
+For every other change pleas create a pull request.
```
Comment (jmesserli) [inline README.md]: please

Comment (needToRoll) [inline README.md]: done

File: README.md (hunk 1 above)
Comment (needToRoll) [inline README.md]: I have added the dots at the end of both sentences

**Pull request merged**